.github/workflows/ci.yml

```
@@ -93,7 +93,14 @@ jobs:
     - name: create project
       run: ./x new template
 
+    # only linux can target linux as it is not clear what that even is exactly
+    # glibc/musl gtk/qt x11/wayland
+    # windows doesn't support posix symlinks so can't cross compile to macos/ios
     - name: build project
-      if: ${{ matrix.target.platform != 'linux' || matrix.host == 'ubuntu-latest' }}
+      if: |
+        !(matrix.host == 'macos-latest' && matrix.target.platform == 'linux' ||
+          matrix.host == 'windows-latest' && matrix.target.platform == 'linux' ||
+          matrix.host == 'windows-latest' && matrix.target.platform == 'macos' ||
+          matrix.host == 'windows-latest' && matrix.target.platform == 'ios')
```

Just [tested](https://github.com/MarijnS95/xbuild/actions/runs/3639528236), this works for me:

```suggestion
      if: >
        !(matrix.host == 'macos-latest' && matrix.target.platform == 'linux' ||
          matrix.host == 'windows-latest' && contains(fromJson('["linux", "macos", "ios"]'), matrix.target.platform))
```

(Click open interesting workflows and see that they had the `build project` step skipped)